.github/issue_template.md

```diff
@@ -25,21 +25,22 @@ Add any other context about the problem here.
 
 -
 
-If this is a "Feature Request", please:
+If this is a "Feature Request", please provide the following information:
 
-Is your feature request related to a problem? Please describe:
-A clear and concise description of what the problem is. 
+Problem description:
+Is your feature request related to a problem? Please provide
+a clear and concise description of what the problem is. 
 
-Describe the solution you'd like:
-A clear and concise description of what you want to happen.
+Solution description:
+Describe the solution you would like. We need a clear and concise description of what you want to happen. Also, have you considered cases outside the expected flow (edge cases)? What should happen in those cases?
 
-Describe alternatives you've considered:
-A clear and concise description of any alternative solutions or features you've considered.
+Description of alternatives
+Describe alternatives you have considered: We need a clear and concise description of any alternative solutions or features you have considered.
```

Comment:
I'd soften the wording here:   If you have considered an alternative, can you provide a ...